src/app/Counter/page.tsx

```diff
@@ -0,0 +1,26 @@
+'use client'
+import React, { useState } from 'react';
+
+const Counter: React.FC = () => {
+  const [count, setCount] = useState<number>(0);
+
+  const increment = () => {
+    setCount(prevCount => prevCount + 1);
+  };
+
+  const decrement = () => {
+    setCount(prevCount => prevCount - 1);
+  };
+
+  return (
+    <div>
+      <button onClick={decrement}>Decrement</button>
+      <br></br>
```

Comment:
Your code is well formatted, but instead of using "br", you should use CSS for spacing and more general styling. I would recommend creating a file such as "globals.css" and then apply the class to your component